test/sync-test.ts

```diff
@@ -79,15 +79,15 @@ describe('napajs/sync', function () {
                 var napa = require('../lib/index');
 
                 napa.store.get('store-for-sync-test-1').set('before-wait', 1);
-                (<any>global).spinWait(100);
+                (<any>global).spinWait(500);
```

Comment:
The fix of sync-test is already merged into master branch.

Reply:
resolved.